### PR TITLE
3.12 blogpost

### PR DESCRIPTION
**Pull request by christinaforney**

### CODE REVIEW

File: blogposts/announcing-sourcegraph-3.12.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+[**🤖 Create draft automation campaigns**](#create-draft-automation-campaigns)<br />
+
+[**🚫 Exclude archived Bitbucket Server repositories in queries**](#exclude-archived-bitbucket-server-repositories-in-queries)<br />
+
+[**⬇️ Download files from code views**](#download-files-from-code-views)<br />
+
+[**🛠 GitHub authentication now supports org membership restriction**](#github-authentication-now-supports-org-membership-restriction)<br />
+
+[**🧪 Experimental features**](#experimental-features)<br />
+Interactive search mode, search across multiple revisions, out of the box repository permissions, faster permissions fetching on Bitbucket Server, and code intelligence precision indicator.
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]: delete period - better to be consistent if it's not a sentence

File: blogposts/announcing-sourcegraph-3.12.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community.
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]: ditto to above

File: blogposts/announcing-sourcegraph-3.12.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+## Create draft automation campaigns
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/386036051?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/386036051" target="_blank">View on Vimeo</a></p>
+</p>
+
+When rolling out changes organization wide, it is desireable to reveiw campaigns with your teammates and to test the change on a specific repository before creating changes across the organization. Automation campaigns can now be created and saved as drafts prior to publishing the campaign and creating changesets (i.e. pull requests) on your code hosts. Changes can be published individually while in draft mode to verify the change with a subset of owners.
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]: *desirable

File: blogposts/announcing-sourcegraph-3.12.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+### Out of the box repository permissions in Sourcegraph
+
+Sourcegraph now provides out of the box repository permissions for teams who have specific customization needs. Teams with multiple code hosts will benefit from setting up access controls natively in Sourcegraph. To enable Sourcegraph repository permissions, add `{ "permissions.userMapping" { "enabled": true, "bindID": "email" } }` to your site configuration.
+
+### Faster permissions fetching on Bitbucket Server
+
+![Faster permissions fetching chart](/images/3-12-permissions.png "Faster permissions fetching chart")
+
+Previously, fetching ACL data from Bitbucket Server was limited by API constraints. Enhancements to the [Sourcegraph Bitbucket Server plugin](https://docs.sourcegraph.com/integration/bitbucket_server#sourcegraph-native-code-intelligence-plugin) have made fetching Bitbucket Server ACL data faster. To take advantage of this improvement install the Bitbucket Server plugin and add `{ "experimentalFeatures": { "bitbucketServerFastPerm": "enabled" } }` to your site configuration.
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]: *To take advantage of this improvement, install

**Comment by christinaforney** [comment]
Suggestions will be added to https://github.com/sourcegraph/about/pull/519

**Pull request merged**